examples/benchmark.rs

```diff
@@ -0,0 +1,26 @@
+#![deny(warnings)]
+use num_cpus;
+use warp::Filter;
+
+const BODY: &str = r#"<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Hello benchmark</title>
+  </head>
+  <body>
+    This is a static content to check the warp performance.
+  </body>
+</html>"#;
+
+fn main() -> std::io::Result<()> {
+    let route = warp::any().map(move || BODY);
+    tokio::runtime::Builder::new_multi_thread()
+        .worker_threads(num_cpus::get())
```

Comment:
If tokio defaults to number of threads then why specify `worker_threads` to number of CPU threads manually?

Reply:
@hamza1311, you are right, thanks for pointing. It was fixed in the next commit. 👍